CoreMapView/src/au/gov/asd/tac/constellation/views/mapview/features/ConstellationAbstractFeature.java

```diff
@@ -36,17 +36,10 @@ public enum ConstellationFeatureType {
     public ConstellationAbstractFeature(Feature feature) {
         this.id = feature.getId();
         switch (feature.getType()) {
-            case POINT:
-                this.type = ConstellationFeatureType.POINT;
-                break;
-            case LINES:
-                this.type = ConstellationFeatureType.LINE;
-                break;
-            case POLYGON:
-                this.type = ConstellationFeatureType.POLYGON;
-                break;
-            default:
-                this.type = ConstellationFeatureType.MULTI;
+            case POINT -> this.type = ConstellationFeatureType.POINT;
+            case LINES -> this.type = ConstellationFeatureType.LINE;
+            case POLYGON -> this.type = ConstellationFeatureType.POLYGON;
+            default -> this.type = ConstellationFeatureType.MULTI;
```

Comment:
this could be changed to move `this.type` out of the switch. i.e.
```
this.type = switch (feature.getType()) {
  case POINT -> ConstellationFeatureType.POINT;
  ...
}
```